assets/tailwind.base.config.js

```diff
@@ -0,0 +1,7 @@
+/*
+ * Add here all the config but the content.
+ */
+module.exports = {
+  theme: {},
+  plugins: [],
+}
```

Comment:
This is the config that will be shared with Beacon Live Admin.

The content property is the only one dynamic, but we don't care about that one in admin, because we are providing our own content.